src/core/components/canvas/static/new-blocks-renderer.tsx

```diff
@@ -191,14 +191,33 @@ const BlocksRenderer = ({
         {(asyncProps) => (
           <BlockRenderer blockAtom={blockAtom} asyncProps={asyncProps}>
             {({ _id, _type, partialBlockId, repeaterItems, $repeaterItemsKey }) => {
-              return _type === "Repeater" ? (
-                isArray(repeaterItems) &&
+              if (_type === "Repeater") {
```

Comment:
We can move this logic inside the RepeaterItem block. Id the children are empty we can show this.
Another important thing is we should pass {...blockProps} to this, so its selectable in builder.